torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -581,6 +581,20 @@ void testRecv(const std::string& path) {
   EXPECT_TRUE(recvCompleted);
 }
 
+void testSetGet(const std::string& path) {
+  const auto size = 2;
+  auto tests = CollectiveTest::initialize(path, size);
+
+  constexpr uint64_t tag = 0x1337;
+  // test that get() gets the same value as the one that was set()
+  auto selfRank = 0;
+  std::vector<uint8_t> testVector = {1, 1, 1, 1};
+  auto store = tests[selfRank].getProcessGroup().getStore();
+  store.set("testKey", testVector);
```

Comment:
Let's test both ranks, i.e. have both ranks `set()` and then the other rank calls `get()`.

Reply:
I believe this is fixed now